scss/_variables.scss

```diff
@@ -38,36 +38,151 @@ $roboto-light: Roboto-Light, $emojifonts, 'Helvetica Neue', Arial, Helvetica, No
 // New colors
 
 html {
```

Comment:
These here act as fallback, but I think about removing them to save a few lines and reduce duplication..